src/app/lib/utilities/srcSet/index.js

```diff
@@ -29,4 +32,7 @@ const createSrcset = (
   return urls.join(', ');
 };
 
-export default createSrcset;
+export const getPlaceholderSrcSet = src =>
```

Comment:
I think we should rethink how to organise this code to ensure it is self-contained and reusable. We now have a function that only exists to support media player placeholders in the `Image` container and this in turn uses a helper from the `MediaPlayer` container, doesn't this create a circular dependency?

Reply:
Yeah it did, moved the files to a utilities folder to avoid circular dependency whereby the utilities have to import files/functions from the containers

Reply:
Yeah it did, moved the files to a utilities folder to avoid circular dependency whereby the utilities have to import files/functions from the containers